extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/CredentialsVerifierExtension.java

```diff
@@ -71,11 +68,6 @@ public JwtCredentialsVerifier createJwtVerifier(ServiceExtensionContext context)
 
     @Provider
     public CredentialsVerifier createCredentialsVerifier(ServiceExtensionContext context) {
-        var hubUrl = context.getSetting(HUB_URL_SETTING, null);
-        if (hubUrl == null) {
-            throw new EdcException(format("Mandatory setting '(%s)' missing", HUB_URL_SETTING));
-        }
-
```

Comment:
nit: just want to know why we are removing this configuration/settings?

Reply:
We forgot to remove it in the last PR